packages/graphback-runtime-knex/src/PostgreDBDataProvider.ts

```diff
@@ -85,8 +93,9 @@ export class KnexDBDataProvider<Type = any, GraphbackContext = any> implements G
   public async findBy(filter: Type | AdvancedFilter, page?: GraphbackPage): Promise<Type[]> {
     const { data } = getDatabaseArguments(this.tableMap, filter);
     //tslint:disable-next-line: await-promise
-    const query = this.db.select().from(this.tableName).where(data);
-    const dbResult = await this.usePage(query, page);
+    // const query = this.db.select().from(this.tableName).where(data);
```

Comment:
I am not sure about the implementation of the `WHERE` clause when the data is in the form of an object.

Reply:
We need to expand parameters with AND